helix-front/client/app/workflow/workflow-detail/workflow-detail.component.html

```diff
@@ -81,7 +81,7 @@
   >
     <mat-button-toggle-group
       #group="matButtonToggleGroup"
-      [value]="workflow.isJobQueue ? 'list' : 'graph'"
+      [value]="'list'"
```

Comment:
The graph view does not consistently render for all inputs, so we set the default view to the list view.